lib/tasks/classroom_management.rake

```diff
@@ -11,7 +11,7 @@ Rails.logger = Logger.new($stdout) unless Rails.env.test?
 # For students to match up the school needs to match with the school defined in profile (hard coded in the helper)
 
 namespace :classroom_management do
```

Comment:
wonder if there's a more appropriate name for this too now, an acronym like CEfE feels weird in a namespace so guess it could be CodeEditorForEducation or something more concise?

Reply:
Maybe just `for_education`? Or is it weird to have a preposition in there? 🤔

Reply:
good call, I think it fits with the frontend page naming nicely and rolls off the tongue a bit better!